### PR TITLE
Porting 4169 to here

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

related: https://github.com/knative/eventing/pull/4168/files

/hold

### CODE REVIEW

**Comment by matzew** [comment]
@aliok based on a slack convo, I port this here

Question is: backport to `eventing-contrib` 0.17 and/or 0.18 ?

**Comment by aliok** [comment]
@matzew we need the same in KafkaSource webhook too: https://github.com/knative-sandbox/eventing-kafka/blob/3b547e91c5511bdda7a0f82aa2cabf713894c023/config/source/deployments/controller.yaml#L63

IMO we should backport this to 0.18 for sure. I also prefer this being backported to 0.17.

File: config/channel/consolidated/deployments/webhook.yaml [hunk 1]
```diff
@@ -66,7 +66,9 @@ spec:
             httpHeaders:
             - name: k-kubelet-probe
               value: "webhook"
-        livenessProbe: *probe
+        livenessProbe:
+          <<: *probe
+          initialDelaySeconds: 20
```
Comment (aliok) [inline config/channel/consolidated/deployments/webhook.yaml:71]: @matzew we need the same in KafkaSource webhook too:

https://github.com/knative-sandbox/eventing-kafka/blob/3b547e91c5511bdda7a0f82aa2cabf713894c023/config/source/deployments/controller.yaml#L63

IMO we should backport this to 0.18 for sure. I also prefer this being backported to 0.17.

Other than that, I verified this fix locally. Fixes the problem.

Comment (matzew) [reply config/channel/consolidated/deployments/webhook.yaml:71]: @aliok +1 on patch 0.17 and 0.18

**Comment by aliok** [comment]
/lgtm
/approve

Thanks

**Comment by aliok** [comment]
I am gonna backport this to eventing-contrib
- master
- 0.18
- 0.17

**Comment by matzew** [comment]
@aliok +1

(not sure on master ...) :-)

**Pull request merged**